support-metrics-common/src/main/java/io/confluent/support/metrics/submitters/ConfluentResponseHandler.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2017 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.support.metrics.submitters;
+
+public interface ConfluentResponseHandler extends ResponseHandler {
```

Comment:
Does this actually need a separate interface or can we extend `ResponseHandler`?

Reply:
After a few days, a separate interface does not look like a good solution anymore. The reason I added it (vs. adding new method to `ResponseHandler` and making all existing clients implement it) was because i did not want to break builds across repos, especially as we are in the middle of the release. So I am going to remove this interface from this PR, and then change ResponseHandler after the CP release. We don't need it right now for C3, because we can  assume JSON responses for C3. We can also replace all error responses in the server to JSON (common code which is also used by KSQL), because none of existing clients (=KSQL and proactive support) check error response right now.  And leave KSQL success response as text for a bit longer.